string state_machine
string IDLE = Idle
string RAIL = Rail
string LAUNCH = Launch
string COAST = Coast
string STOP = Stop

float64 time_now